stacks_queues/queue.py

```diff
@@ -23,39 +23,58 @@ def enqueue(self, element):
             In the store are occupied
             returns None
         """
-        pass
+        if self.size == INITIAL_QUEUE_SIZE:
```

Comment:
I would suggest using `buffer_size` over `INITIAL_QUEUE_SIZE` here. Imagine you alter your implementation so that the `buffer_size` could expand in certain cases. Then you would have to manually change all of your references to `INITIAL_QUEUE_SIZE` to maintain your code. 
```suggestion
        if self.size == self.buffer_size:
```